package.json

```diff
@@ -2,14 +2,19 @@
   "name": "iopipe",
   "version": "0.3.1",
   "description": "IOpipe Lambda Analytics & Tracing Agent",
-  "main": "index.js",
+  "main": "dist/iopipe.js",
   "scripts": {
-    "test": "istanbul cover jasmine --captureExceptions"
+    "test": "istanbul cover jasmine --captureExceptions",
+    "build": "webpack --progress --profile --colors",
+    "prepublish": "webpack --progress --profile --colors"
```

Comment:
when you do `npm install` inside the iopipe directory? or from a package that depends on iopipe?

I tested /w npm 3 and had a few quirks, but npm 5 is fine /w the following process:

```
$ git clone https://github.com/iopipe/iopipe
$ cd iopipe
$ npm install
$ mkdir ~/project
$ cd ~/project
$ npm init -f
$ npm install ~/iopipe
```

It is necessary to do `npm install` inside the iopipe source directory _before_ doing `npm install` from `~/project`. I spoke to friends at npm and didn't get a better answer.